dists/scummvm.6

```diff
@@ -14,226 +13,423 @@
+Display list of supported engines and exit.
+.It Fl -list-saves
+Display a list of saved games for the target specified with
+.Fl -game= Ns Ar TARGET
+or all targets if none is specified.
+.Fl -game= Ns Ar ID
+In combination with
+.Fl -add
+or
+.Fl -detect
+only adds or attempts to detect the game with id
+.Ar ID .
+.It Fl -list-themes
+Display list of all usable GUI themes.
+.It Fl -md5
+Shows MD5 hash of the file given by
+.Fl -md5-path= Ns Ar PATH .
+If
+.Fl -md5-length= Ns Ar NUM
+is passed then it shows the MD5 hash of the first or last
+.Ar NUM
+bytes of the file given by
+.Ar PATH .
+If
+.Fl -md5-engine= Ns Ar ENGINE_ID
+is passed, it fetches the MD5 length automatically, overriding
+.Fl -md5-length .
+.It Fl -md5mac
+Shows MD5 hash for both the resource fork and data fork of the
+mac file given by
+.Fl -md5-path= Ns Ar PATH .
+If
+.Fl -md5-length= Ns Ar NUM
+is passed then it shows the MD5 hash of the first or last
+.Ar NUM
+bytes of each fork.
+.It Fl -midi-gain= Ns Ar NUM
+Set the gain for MIDI playback, 0-1000 (default: 100).
+.br
+(only supported by some MIDI drivers)
 .It Fl -multi-midi
 Enable combination of AdLib and native MIDI.
 .It Fl -native-mt32
 True Roland MT-32 MIDI (disable GM emulation).
-.It Fl -enable-gs
-Enable Roland GS mode for MIDI playback.
+.It Fl -no-filtering
+Force unfiltered graphics mode.
+.It Fl -opl-driver= Ns Ar DRIVER
+Select AdLib (OPL) emulator
+.Po
+.Ar db ,
+.Ar mame ,
+.Ar nuked
+.Pc .
 .It Fl -output-rate= Ns Ar RATE
 Set output sample rate in Hz (e.g. 22050).
-.It Fl -opl-driver= Ns Ar DRIVER
-Select AdLib (OPL) emulator (\fIdb\fR, \fImame\fR, \fInuked\fR)
-.It Fl -aspect-ratio
-Enable aspect ratio correction.
+.It Fl -platform= Ns Ar WORD
+Specify platform of game
+.Po
+.Ar 2gs ,
+.Ar 3do ,
+.Ar acorn ,
+.Ar amiga ,
+.Ar android ,
+.Ar apple2 ,
+.Ar atari ,
+.Ar atari8 ,
+.Ar beos ,
+.Ar c64 ,
+.Ar cdi ,
+.Ar coco ,
+.Ar coco3 ,
+.Ar fmtowns ,
+.Ar ios ,
+.Ar linux ,
+.Ar macintosh ,
+.Ar macintosh2 ,
+.Ar megadrive ,
+.Ar nes ,
+.Ar os2 ,
+.Ar pc ,
+.Ar pc98 ,
+.Ar pce ,
+.Ar pippin ,
+.Ar playstation ,
+.Ar playstation2 ,
+.Ar ppc ,
+.Ar saturn ,
+.Ar segacd ,
+.Ar shockwave ,
+.Ar ti994 ,
+.Ar wii ,
+.Ar windows ,
+.Ar xbox ,
+.Ar zx
+.Pc .
+.It Fl -recursive
+In combination with
+.Fl -add
+or
+.Fl -detect
+recurse down all subdirectories.
 .It Fl -render-mode= Ns Ar MODE
-Enable additional render modes (\fIhercGreen\fR, \fIhercAmber\fR, \fIcga\fR,
-\fIega\fR, \fIvga\fR, \fIamiga\fR, \fIfmtowns\fR, \fIpc9821\fR, \fIpc9801\fR,
-\fI2gs\fR, \fIatari\fR, \fImacintosh\fR)
-.It Fl -alt-intro
-Use alternative intro for CD versions of Beneath a Steel Sky and Flight of the
-Amazon Queen.
-.It Fl -copy-protection
-Enable copy protection in games, when ScummVM disables it by default.
+Enable additional render modes
+.Po
+.Ar hercGreen ,
+.Ar hercAmber ,
+.Ar cga ,
+.Ar ega ,
+.Ar vga ,
+.Ar amiga ,
+.Ar fmtowns ,
+.Ar pc9821 ,
+.Ar pc9801 ,
+.Ar 2gs ,
+.Ar atari ,
+.Ar macintosh
+.Pc .
```

Comment:
```suggestion
.Po
.Ar hercGreen ,
.Ar hercAmber ,
.Ar cga ,
.Ar cgaComp ,
.Ar cgaBW ,
.Ar ega ,
.Ar vga ,
.Ar amiga ,
.Ar fmtowns ,
.Ar pc9821 ,
.Ar pc9801 ,
.Ar 2gs ,
.Ar atari ,
.Ar macintosh ,
.Ar macintoshbw
.Pc .
```

from:

https://github.com/scummvm/scummvm/blob/1c7e799f510c6734eaaecda7ee76d3f0963725c1/common/gui_options.cpp#L63-L77

(commandLine.cpp and doc/ seem out-of-date too… will provide a separate PR for that)